guess.js

```diff
@@ -114,6 +120,8 @@ function guessCheck() {
 
     let successguess = true;
     for (let i = 1; i <= numbersOfLeters; i++) {

```

Comment:
Replying to [this](https://github.com/mohamed-sabry-1789/guess-word-game/pull/1/files#r1536675843).
Yes, exactly.
```suggestion
    for (let i = 1; i <= wordGuess.length; i++) {

```
You'll also need to use the same condition in `guessCheck` function below.
I suggest making a const for this for example: `const wordLength = wordGuess.length` and reuse it in both functions, and anywhere else that references `numbersOfLeters`, and then remove it.

Reply:
but i use numberOfLetters before i build arry to words 

how can i used it to build my input field

Reply:
Yes, good point. But you already initialise the wordGuess before you call this function so it's still in the global scope.
I would be more specific and create a function to initialiseWord, and pass the word to `genrateInput` function and `guessCheck` function so that each function is more specific and doesn't need to access any variables from the global scope.